admin/class-admin-asset-manager.php

```diff
@@ -188,23 +196,23 @@ private function scripts_to_be_registered() {
 				// Load webpack-commons for bundle support.
 				'src'  => 'commons-' . $flat_version,
 				'deps' => array(
-					self::PREFIX . 'babel-polyfill',
+					$this->prefix . 'babel-polyfill',
```

Comment:
Please change all occurrences of `$this->prefix` to `self::PREFIX` in this method, because they belong to yoast-seo.